pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/controller/PinotControllerTransport.java

```diff
@@ -77,13 +76,31 @@ public SchemaResponse getTableSchema(String table, String controllerAddress) {
       final Future<Response> response =
           requestBuilder.addHeader("Content-Type", "application/json; charset=utf-8").execute();
 
-      SchemaResponseFuture schemaResponseFuture = new SchemaResponseFuture(response, url);
+      SchemaResponse.SchemaResponseFuture schemaResponseFuture = new SchemaResponse.SchemaResponseFuture(response, url);
       return schemaResponseFuture.get();
     } catch (ExecutionException e) {
       throw new PinotClientException(e);
     }
   }
 
+  public ControllerTenantBrokerResponse getBrokersFromController(String controllerAddress, String tenant) {
```

Comment:
is this called for every connection creation or every query?

Reply:
every connection creation.

Reply:
Resolved in #6020